### DISCUSSION TITLE
A newbie's question

### DISCUSSION THREAD

**Discussion by jinserk** [apache/airflow#21593] [Q&A]
Hi all,

I just start to learn about Airflow. My question is, can the KubernetesExecutor or KubernetesPodExecutor run any docker image that doesn't have airflow installed inside? 

Thank you!

**Comment by potiuk** [comment]
Yes.

**Comment by jinserk** [comment]
@potiuk Thank you for the prompt answer. I've tried to install Airflow into k8s with KubernetesExecutor but when I run an example dag, it doesn't invoke any pod for the task -- so I suspect the image which doesn't have airflow inside.

**Reply by potiuk** [reply]
No idea. It shoudl work. Jus follow  the Helm Chart https://airflow.apache.org/docs/helm-chart/stable/index.html and quick start and examples and Airflow should work.
